src/index.js

```diff
@@ -1 +1,52 @@
-import './style.css'
\ No newline at end of file
+import './style.css';
+import tasksHtml from './modules/innerhtml.js';
+
+const form = document.querySelector('.form');
+const inputForm = document.querySelector('.input-form');
+const allTasks = document.querySelector('.tasks');
+
+const todoTasks = [];
```

Comment:
- [ ] Kindly make the `todoTasks` contain some static object, each object has:
       - description: string
       - completed: boolean
       - index: number
as in the requirement, so it will be like:
```
const todoTasks = 
[
  {
    description: 'Task 1',
    completed: false,
    index: 1,
  },
  {
    description: 'Task 2',
    completed: false,
    index: 2,
  },
]
```
then the list is rendered when the page is loaded.